### PR TITLE
Add footer component

### LINKED ISSUE #92: Add Footer to Vocabulary
We need to add the Footer to Vocabulary to match the Figma DL: https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=639%3A24

### PR DESCRIPTION
**Pull request by hugosolar**
**Fixes**
Fixes #92

**Description**
This PR add the footer component but it's still incomplete because we still doesn't have the icons library so the social media Icons section and CC icons are still pending.

**Screenshots**
Screen Shot 2020-03-17 at 11 05 55 AM

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

File: src/stories/footer.stories.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { svgCode } from './logos.stories.mdx'
+
+export default { title: 'Footer' }
+
+export const Main = () => {
+  return `
+    <footer class="main-footer">
+      <div class="container">
```
Comment (dhruvkb) [inline src/stories/footer.stories.js:8]: If the footer uses the container, the header must have it too for vertical alignment. I mean it in the context of [this comment](https://github.com/creativecommons/vocabulary/pull/176#discussion_r393009240).

**Review by dhruvkb — COMMENTED** [review]
Just one thing I find to be strange about this.

File: src/stories/footer.stories.js (hunk 1 above)
Comment (hugosolar) [reply src/stories/footer.stories.js:8]: Yes... I agree it's necessary for alignment

**Review by dhruvkb — APPROVED** [review]
Looks awesome to me!

**Comment by hugosolar** [comment]
@dhruvkb I realize we're going to have the same problem with the navbar component from Bulma, I'll make the changes to avoid using it and let you know

File: src/stories/footer.stories.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            </a>
+            <p>
+              <address class="margin-bottom-normal"> Creative Commons <br> PO Box 1866, Mountain View CA 94042</address>
+              <a href="mailto:info@creativecommons.org" class="mail">info@creativecommons.org</a>
+              <a href="tel://+1-415-429-6753" class="phone">+1-415-429-6753</a>
+            </p>
+          </div>
+          <div class="column is-half">
+            <nav class="footer-navigation navbar">
+              <ul class="navbar-menu is-active">
```
Comment (dhruvkb) [inline src/stories/footer.stories.js]: I didn't see the `navbar-menu` here.

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**